python/cudf/cudf/_lib/merge.pyx

```diff
@@ -10,79 +10,39 @@ from cudf._lib.column cimport Column
 from cudf._lib.cpp.merge cimport merge as cpp_merge
 from cudf._lib.cpp.table.table cimport table
 from cudf._lib.cpp.table.table_view cimport table_view
-from cudf._lib.utils cimport data_from_unique_ptr, table_view_from_table
+from cudf._lib.utils cimport columns_from_unique_ptr, table_view_from_columns
 
 
 def merge_sorted(
-    object tables,
-    object keys=None,
-    bool by_index=False,
-    bool ignore_index=False,
+    list list_of_columns,
```

Comment:
Perhaps

```suggestion
    list input_columns,
```

?

Reply:
Actually, I'm trying to differentiate this with other cython APIs where we are expecting `List[List[Column]]` here, not just `List[Column]`. I'm still fond of `input_columns` but I'll add an informal docstring to describe this better.